Makefile

```diff
@@ -709,3 +709,30 @@ shunit2: common-test #TODO include other tests
 .PHONY: common-test
 common-test:
 	./scripts/common/test/common-test.sh
+
+.PHONY: init-sbom
+init-sbom:
+	mkdir -p sbom/spdx sbom/assets
+	# remove directory that breaks go sbom generation 
+	- rm -rf web/node_modules/snyk/dist/gosrc
+	- rm -rf web/node_modules/snyk-go-plugin/gosrc
+
+.PHONY: install-spdx-sbom-generator
+install-spdx-sbom-generator: init-sbom
+ifeq (,$(shell command -v spdx-sbom-generator))
+	./scripts/initialize-build.sh
+SPDX_GENERATOR=./sbom/spdx-sbom-generator
+else
+SPDX_GENERATOR=$(shell command -v spdx-sbom-generator)
+endif
+
+.PHONY: generate-sbom
+generate-sbom: install-spdx-sbom-generator 
+	$(SPDX_GENERATOR) -o ./sbom/spdx        
```

Comment:
One thing I don't like about spdx-sbom-generator is it will build sboms for everything it finds in the current directory.  I'm thinking about submitting a PR to them to change that.  It outputs different files for the different language packages it analyzes and the only way you can be selective is to choose the files that you want to include.  

The other missing piece here is sboms for images, but that's for a different pr